src/codegen/riscv64/register-riscv64.h

```diff
@@ -36,6 +36,12 @@ namespace internal {
   V(w16) V(w17) V(w18) V(w19) V(w20) V(w21) V(w22) V(w23)  \
   V(w24) V(w25) V(w26) V(w27) V(w28) V(w29) V(w30) V(w31)
 
+#define VECTOR_REGISTERS(V)                               \
```

Comment:
Since we define VECTOR_REGISTERS, definition of the SIMD128_REGISTERS can be removed. Also, all use of the SIMD128_REGISTERS can be changed to VECTOR_REGISTERS.